README.md

```diff
@@ -65,8 +65,6 @@ The editor has some special features:
 
 The XML autocompletion is custom made for the Frank!Flow and is made available for everybody that wants XML completion based on XSD’s for the Monaco Editor. Like this project, it is open-source and can be found [here on Github](https://github.com/philipsens/monaco-xsd-code-completion).
 
-The autocompletion has support for multiple XSD’s, namespaces and isn’t just limited to the IbisDoc. Just place the XSD in the configuration folder and reference it in the configuration to add autocompletion.
-
```

Comment:
I would add this to the README of Sergi's Monaco code completion project.

Reply:
Yes, maybe in the future we can exp lean how the references should be made.